adapters/triplelift_native/triplelift_native_test.go

```diff
@@ -0,0 +1,14 @@
+package triplelift_native
+
+import (
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"testing"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bar := NewTripleliftNativeBidder(nil, "", "{foo:2}")
```

Comment:
Sorry for missing this earlier but this seems like a separate test case that doesn't really test the json samples you have in the `triplelift_nativetest` directory. So, I'd recommend moving this test case into it's own function and also, rather than panicking on L11, please consider using the `assert` function in the `testify/assert` library

Reply:
ok, I've broken it out into its own test, and used the assert library